static/js/main.js

```diff
@@ -58,29 +58,26 @@ function onSettingsLinkClick() {
 }
 
 function getDurationText(durationSeconds) {
-  const secondsInMinute = 60;
-  let secondText = "second";
+  const minutes = Math.floor((durationSeconds % 3600) / 60);
+  const remainingSeconds = Math.floor(durationSeconds % 60);
 
-  if (durationSeconds == 0) {
-    return "";
-  }
-  if (durationSeconds == 1) {
-    return joinText([durationSeconds, secondText]);
-  }
+  let minuteStr
+  let secondStr
 
-  if (durationSeconds < secondsInMinute) {
-    secondText += "s";
-    return joinText([durationSeconds, secondText]);
+  if (minutes > 0) {
+    minuteStr = minutes + "m "
+  } else {
+    minuteStr = ""
   }
 
-  let minuteText = "minute";
-  const durationMinutes = Math.floor(durationSeconds / 60);
-  if (durationMinutes == 1) {
-    return joinText([durationMinutes, minuteText]);
+  if (remainingSeconds != 0) {
+    secondStr = remainingSeconds + "s"
```

Comment:
in the code you removed  there is logic to work with long words like minute(s) and second(s). When you remove a code you should not change the logic just to make your life easier. 

Such changes in logic should be first agreed with the project manager or code maintainer.